Annotato/Annotato/Views/Document/Edit/DocumentView.swift

```diff
@@ -26,6 +27,14 @@ class DocumentView: UIView {
         }
     }
 
+    private func initializePdfView() {
+        let view = DocumentPdfView(
+            frame: self.frame,
```

Comment:
On my screen, the pdf exceeds the bounds on the right and bottom sides, is it possible to fix it?
![photo_2022-03-17 10 30 35](https://user-images.githubusercontent.com/71375383/158724601-eaa752f8-254c-48b8-ae60-b2e5e38b337e.jpeg)

Reply:
Ok I will explore more about this first and resolve it in another PR